<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>digitalLibrary</class>
 <widget class="QDialog" name="digitalLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="manageAuthorButton">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>60</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Manage Authors</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteMemberBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>100</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Delete Member</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>10</y>
        <width>565</width>
        <height>39</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; font-style:italic; color:#005500;&quot;&gt;Members&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addBookBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>130</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Add Book</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="editMemberBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>160</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Edit Member</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addMemberBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>190</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Add Member</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="statsButton">
      <property name="geometry">
       <rect>
        <x>-170</x>
        <y>220</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Statistic</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="membersListBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>250</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Members List</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteBookBtn">
      <property name="geometry">
       <rect>
        <x>-170</x>
        <y>280</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Delete Book</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="booksListBtn">
      <property name="geometry">
       <rect>
        <x>-160</x>
        <y>340</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Books List</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>440</y>
        <width>151</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/login-book.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>565</width>
        <height>49</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; font-style:italic; color:#005500;&quot;&gt;Books&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>565</width>
        <height>54</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(85, 170, 0)qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0))</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; font-style:italic; color:#005500;&quot;&gt;Authors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-250</x>
        <y>0</y>
        <width>565</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 18pt &quot;Georgia&quot;;
color:  rgb(210, 156, 255);/* Violet foncé */
text-shadow: 1px 1px 2px #FFD70033;  /* Ombre dorée subtile */</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:700; color:#005500;&quot;&gt;Welcome Back&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>565</width>
        <height>69</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 85, 0)</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; font-style:italic; color:#005500;&quot;&gt;Genres&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>565</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 18pt &quot;georgia&quot;;
color:black;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Books</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="editBookBtn">
      <property name="geometry">
       <rect>
        <x>-170</x>
        <y>310</y>
        <width>565</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:rgb(255, 168, 110);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Edit Book</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>565</width>
        <height>96</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 18pt &quot;georgia&quot;;
color: black;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Members</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="memberNum">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>300</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 24pt &quot;Georgia&quot;;
color: black;
text-shadow: 0 0 5px #FFD700;  /* Lueur dorée */</string>
      </property>
      <property name="text">
       <string>22</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>565</width>
        <height>147</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 18pt &quot;georgia&quot;;
color:black;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Authors</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="bookNum">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>321</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 24pt &quot;Georgia&quot;;
color: black;
text-shadow: 0 0 5px #FFD700;  /* Lueur dorée */</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="authorNum">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 24pt &quot;Georgia&quot;;
color:black;
text-shadow: 0 0 5px #FFD700;  /* Lueur dorée */</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
